src/amo/reducers/errors.js

```diff
@@ -6,6 +6,11 @@ import {
   ERROR_UNKNOWN,
 } from 'amo/constants';
 import log from 'amo/logger';
+import { sanitizeHTML } from 'amo/utils';
+
+const sanitizeMessage = (message) =>
+  sanitizeHTML(typeof message === 'object' ? JSON.stringify(message) : message)
```

Comment:
From the test results this seems a little extreme - some errors may legitimately contain html tags because the input contained html tags (I guess? I've not looked for examples, but it's possible) - this just strips the tags entirely which could result in a confusing error.  Is there an option to escape the html instead so it's still displayed to the user, but safely?

Reply:
My thought here was that we really never should have html in an error message, and if we do it is likely the result of the user trying to do something suspicious. I'm not concerned about simply stripping all HTML out of any error messages that we need to display back to the user. Maybe, if we're worried about debugging, we could log the pre-stripped output so we can see it in the logs if necessary?

Reply:
I (personally) wasn't worried about debugging, just error messages of the style `"<script>baa</script>" isn't valid` being rendered to the user as  `"" isn't valid`.  `sanitizeHTML` appears to default to not allowing _any_ tags either, so it'd strip out harmless tags like `<b>` and other tags we allow in certain input fields.  But that said, I don't have a concrete example of where the API does this.

Reply:
I am aware of the ability to not strip out certain tags when using `sanitizeHTML`, but chose not to use that. I am working on the assumption that we will never intentionally include HTML in an error message that we generate, and if that is the case then I think it's fine to simply strip out everything.  Yes, your example above is possible, but seems unlikely, and in the case where someone did intentionally try to inject HTML where they shouldn't have, I have no problem with them seeing an error message that doesn't make complete sense.

Are we okay to ship this as is, or would you prefer I allow some common tags and/or change it to escape HTML rather than strip it? I think the nice thing about `sanitizeHTML` is that it's well tested, and doesn't involve us trying to manually deal with strings in a way that an attacker could potentially get around.

Reply:
I've just done a quick look and couldn't find anywhere in the api we return user input in an error message for a field that could legitimately have html tags so in the absence of a concrete example of breakage lets not complicate things - it's fine, it's fine.

Reply:
FWIW I also prefer this, IMHO we should not voluntarily include HTML in our error messages, that's up to the client to do that.